src/stories/Forms/Select/Option.tsx

```diff
@@ -25,29 +26,42 @@ const Wrapper = styled.li`
     flex-shrink: 0;
   }
 
-  & > *:first-child:is(figure, img, svg) {
+  & > *:first-child:is(figure, img, svg, .checkbox) {
     margin-left: ${rem(-4)};
   }
 
-  & > * {
-    pointer-events: none;
+  .checkbox {
```

Comment:
I think we should stay to the Styled component syntax so do StyledCheckbox etc here:)